eng/pipelines/templates/steps/generate-doc.yml

```diff
@@ -0,0 +1,17 @@
+steps:
+  - script: |
+      npm i -g typedoc typescript
+    displayName: "Install typedoc & typescript"
+
+  - script: |
+      npm install
```

Comment:
```npm ci``` is safer and sometimes faster.

Reply:
will follow up on this in a separate PR and add lock files for all JS tools